cypher-shell/src/main/java/org/neo4j/shell/CommandReader.java

```diff
@@ -0,0 +1,41 @@
+package org.neo4j.shell;
+
+import jline.console.ConsoleReader;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommandReader {
+    private final ConsoleReader reader;
+    private final Shell shell;
+    static final Pattern MULTILINE_BREAK = Pattern.compile("\\\\\\s*$");
```

Comment:
In not escaped form it's originally `\\\s+$` right? and we try to catch only one slash and {spaces} in the end? if so, it looks fine, i thought that we need to match '\\'. Please correct me if i missing something